PIM/API/01_Overview.md

```diff
@@ -0,0 +1,227 @@
+
+
+### Tools
+
+You can use different software tools to submit your requests, either your own tool or a third-party tool, such as Postman. For a detailed description on how to set up Postman, see [Set up Postman for the Actindo Core1 OpenAPI](#set-up-postman-for-the-actindo-core1-openapi). 
+
+Regardless of how you submit your requests, you always need to provide the following information:
+
+- URI
+- HTTP method 
+- Headers
+- Request body
+
+
+## Set up Postman for the Actindo Core1 OpenAPI
+
+[comment]: <> (Imogen: Das ist ein Versuch zu deinem Vorschlag, dass diese Info unter Overview/Setup kommen sollte. Besser so oder in einer separaten Datei? Trotzdem weiß ich nicht, ob das bleibt, da Julian gesagt hat, es wäre nicht unbedingt nötig... Daher habe ich auch separat woanders geparkt.)
+
+To send requests to the *Actindo Core1 Platform* API endpoints, you can use an external API tool, such as Postman. For detailed information on download, configuration, and basic functions, see the [Postman](https://www.postman.com/ "[https://www.postman.com/]") website.
+
+Before you can send requests to the *Actindo Core1 OpenAPI* with Postman, you have to set it up to interact with your system.  
+
+
+### Define the headers
+
+Headers must be defined, so that the data is transferred to the *Actindo Core1 OpenAPI* in the appropriate format and responses are transmitted back to the user. To do so, follow the procedure below:
+
+1. Click the *Headers* tab. 
+
+2. Enter "Content-Type" and "Accept" in two different rows the *Key* column. 
+
+3. Enter "application/json" in the *Value* column for both rows. This setting specifies that the data being sent in the request body is in JSON format.
+
+
+### Set the session cookie
+
+Before you can communicate with the Core 1, you have to set a session cookie. To do so, follow the procedure below:
+
+1. Send a request via the [Send] button.  
+    You receive the following response:
+
+        {
+        "success": false,
+        "errno": 10000,
+        "error": "Not  logged in",
+        "trace": ""
+        }
+    
+    This response indicates that the request failed because you are not logged into your *Actindo Core1 Platform* instance in Postman. The API request must include the session ID from the *Actindo Core1 Platform* instance in which you are currently logged in.
+
+2. Switch to the browser where you are logged into your *Actindo Core1 Platform* instance. 
+
+3. Open the console in the browser developer tools. Depending on the browser you use, the access shortcut or key may differ. For example, in Chrome you can access the console pressing the **F12** key.
+
+4. In the browser developer tools console, click the *Application* tab and expand the *Cookies* option in the left menu under the *Storage* section.  
+    The URL of your current *Actindo Core1 Platform* instance is displayed.
```

Comment:
Core1 PLatform vs. Actindo Core1 Platform ?? -> Wie wollen wir es einheitlich nennen? In Core1 Doku bin ich da leider auch nicht  einheitlich..